sigs.yaml

```diff
@@ -958,10 +958,16 @@ sigs:
       day: Wednesday
       time: "9:30"
       tz: "PT (Pacific Time)"
-      frequency: weekly
+      frequency: weekly - except fourth Wednesday every month
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/
       recordings_url: https://www.youtube.com/watch?v=EMGUdOKwSns&list=PL69nYSiGNLP2x_48wbOPO0vXQgNTm_xxr
+    - description: APAC SIG Meeting
+      day: Wednesday
+      time: "20:00"
```

Comment:
Is this 8pm or 9pm? DST has me confused. :thinking: 

/cc @castrojo @mrbobbytables 
our timezone experts :)

Reply:
PT is like a soft reference that moves with DST. It'd "always" be 8pm for either PST or PDT.

I wish we'd get rid of daylight savings time -_-

Reply:
Thanks!

India doesn't use DST, so when I checked with the meeting time from last month, I got confused. :upside_down_face:

Reply:
I don't blame ya for the confusion XD 

It's so incredibly frustrating.

Reply:
`PT` changed from `PST` to `PDT` automatically, so I guess our next meeting will be one hour earlier than the last one for non-DST timezones while it's the same time for PT folks.

Reply:
I did look at https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#dates-and-times when I was adding this 😄